src/csi-wrapper/pkg/wrapper/controllerservice.go

```diff
@@ -67,7 +67,7 @@ func NewControllerService(targetEndpoint, namespace string, peerpodvolumeClientS
 }
 
 func (s *ControllerService) redirect(ctx context.Context, req interface{}, fn func(context.Context, csi.ControllerClient)) error {
-	conn, err := grpc.Dial(s.TargetEndpoint, grpc.WithBlock(), grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
hmm  afaik we don't have automated tests for the csi-wrapper component. How to test it doesn't introduce a regression?

Reply:
I'm not familiar with the csi-wrapper , so not sure. I just followed the documentation with the new way to do. I tried the code in a standalone method as part of a hello world example, not as part of the csi-wrapper.